src/System.IdentityModel.Tokens/LogMessages.cs

```diff
@@ -19,9 +19,9 @@
 namespace System.IdentityModel.Tokens
 {
     /// <summary>
-    /// Error codes and messages
+    /// Log messages and codes
```

Comment:
That was in the initial plan for this PR. But some messages are overlapping across different projects. That can result in some redundancy. Though I created a separate one for M.IM.Logging since that cannot take a dependency on S.IM.Tokens. I will try to separate them out, that's better than having it all in one.